Dockerfile

```diff
@@ -43,45 +46,44 @@ RUN apt-get -qq -y update \
  && rm -rf /opt/geos* \
 ## Protobuf
  && cd /opt/ \
- && curl -L https://github.com/google/protobuf/archive/v3.0.2.tar.gz | tar xvz && cd protobuf-3.0.2 \
+ && curl -L https://github.com/google/protobuf/archive/v$PROTOBUF_VERSION.tar.gz | tar xvz && cd protobuf-$PROTOBUF_VERSION \
  && ./autogen.sh \
  && ./configure \
  && make \
  && make install \
  && ldconfig \
- && rm -rf /opt/protobuf-3.0.2 \
+ && rm -rf /opt/protobuf-$PROTOBUF_VERSION \
 ## Protobuf-c
  && cd /opt/ \
- && curl -L https://github.com/protobuf-c/protobuf-c/releases/download/v1.2.1/protobuf-c-1.2.1.tar.gz | tar xvz && cd protobuf-c-1.2.1 \
+ && curl -L https://github.com/protobuf-c/protobuf-c/releases/download/v$PROTOBUF_C_VERSION/protobuf-c-$PROTOBUF_C_VERSION.tar.gz | tar xvz && cd protobuf-c-$PROTOBUF_C_VERSION \
  && ./configure \
  && make \
  && make install \
  && ldconfig \
- && rm -rf /opt/protobuf-c.1.2.1 \
+ && rm -rf /opt/protobuf-c-$PROTOBUF_C_VERSION \
 ## Postgis
  && cd /opt/ \
- && git clone -b svn-trunk https://github.com/postgis/postgis.git \  
- && cd postgis \
- && git reset --hard ff0a844e606622f45841fc25221bbaa136ed1001 \ 
+ && curl -L https://download.osgeo.org/postgis/source/postgis-$POSTGIS_VERSION.tar.gz | tar xvz && cd postgis-$POSTGIS_VERSION \
  && ./autogen.sh \
  && ./configure CFLAGS="-O0 -Wall" \
```

Comment:
If you are interested in benchmarking the postgis performance, you may also want to try and remove the `-O0` here (or replace it with `-O3`?), and see if things are still stable. My guess would be that it was added to work around some compiler bug (?), which _should_ not be relevant anymore